package.json

```diff
@@ -32,7 +32,6 @@
   "snyk": true,
   "devDependencies": {
     "@types/marked": "^0.6.5",
-    "@types/minimist": "^1.2.0",
```

Comment:
how the types for minimist are dropped?

Reply:
I've also removed `minimist` all together as it is not being mantained for almost 5 years